### ISSUE TITLE
Not downloading better quality

### ISSUE THREAD

**Issue by joakim93** [SickRage/SickRage#2968]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit:Branch: master
Commit: a3ce318f2c3b207da474db4c8e3cf62bb9032e0e
Database Version: 44.0
OS: Freenas 9.10
What you did: Forced backlog search on game of thrones season 1
What happened: didn´t download anything
What you expected: That my 1 and second episode wouls had been downloaded in better quality
My quality settings: Allowed: SDTV, SD DVD, 720p HDTV, 1080p HDTV, 720p WEB-DL, 1080p WEB-DL, 720p BluRay, 1080p BluRay, 4K UHD TV, 4K UHD WEB-DL, 4K UHD BluRay, 8K UHD TV, 8K UHD WEB-DL, 8K UHD BluRay
Preferred: 8K UHD BluRay
Logs:
```
https://gist.githubusercontent.com/joakim93/6a8d42be7cdcaad50005b86ea18dc070/raw/1956faa7398815c415b7e267719aeed9bc162dff/sickrage.log
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by beschoenen** [comment]
SickRage will only replace your episodes if it can find the episodes in one of your preferred qualities. Since it's prob unable to find 8K Game of Thrones it won't do anything. Try setting your preferred quality to 1080p WEB-DL and/or BluRay.

**Comment by joakim93** [comment]
Okay I understand that but why doesn't it gradually update quality? How do I set it so I have the first episodes that are available but when a better quality comes out I get that.

**Comment by beschoenen** [comment]
It does exactly that, but only once, and only if a quality in your preferred list is available. Game of Thrones is not available in 8K so it won't be able to find that.

**Comment by joakim93** [comment]
So there is no way to have the best quality downloaded automatically? In WIKI it says : I'm an Archivist, give me it all!
Use the Custom setting to choose the Allowed Quality of SD, and choose every other Quality under the Preferred Qualities. As a fair warning: This will download every release that comes out for that show in order of increasing quality, as they are released.
So I tried this and now all my game of thrones season 1 is 720p bluray. Before just episode 1 and 2 was sdtv and the rest were 1080p bluray. 
I just want to have the first quality available and that it progressively downloads better quality.

**Comment by beschoenen** [comment]
@miigotu Am I wrong? SickRage only downloads 1 preferred quality file, right?

**Comment by miigotu** [comment]
It was changed because of people doing exactly that, and spamming the shit out of providers for thousands of episodes that will NEVER reach higher qualities.

Put the lower qualities in allowed, and your max wanted quality in the preferred, since it will stop on the first preferred quality.
